openedx/adg/lms/webinars/models.py

```diff
@@ -101,7 +101,8 @@ def to_dict(self):
             'webinar_id': self.id,
             'webinar_title': self.title,
             'webinar_description': self.description,
-            'webinar_start_time': self.start_time.strftime(WEBINARS_TIME_FORMAT),
+            'webinar_start_time': convert_time_zone(self.start_time, WEBINAR_DEFAULT_TIME_ZONE,
+                                                    WEBINAR_DATE_TIME_FORMAT),
```

Comment:
Pls follow edX guidelines for line break, Break after the opening parenthesis and the closing parenthesis should come on the new line.